conceptual/Npgsql/release-notes/8.0.md

```diff
@@ -1,29 +1,33 @@
 # Npgsql 8.0 Release Notes
 
-Npgsql version 8.0 is under development, and is available as preview versions.
+**The below release notes and breaking changes list aren't yet complete. Check back soon for more notes.**
+
+Npgsql version 8.0 is under development, and is available as release candidate versions.
 
 ## New features
 
-## Breaking changes
+### NativeAOT and trimming support
+
+Npgsql 8.0 now has 1st-class support for NativeAOT and trimming; the entire library has been properly annotated and is safe for use in applications. The majority of features have been made compatible with NativeAOT/trimming and can be used without issues, and most applications using Npgsql can be used as-is with NativeAOT/trimming without any changes. A few features which are incompatible require an explicit code opt-in, which generates a warning if used with NativeAOT/trimming enabled ([see breaking change note](#dynamic-optin)).
 
-### System.Text.Json mapping support must now be opted into
+Considerable effort has gone into reducing Npgsql's size footprint; a minimal Npgsql application using NativeAOT and trimming now takes only around 5MB of disk space. To allow users to achieve a minimal size footprint, <xref:Npgsql.NpgsqlSlimDataSourceBuilder> has been introduced; unlike the standard <xref:Npgsql.NpgsqlDataSourceBuilder>, this builder includes only the very minimum of functionality by default, and allows adding additional features via opt-ins. This allows a pay-per-play approach to application size, where developers can choose only the features they actually need for optimal size. For more information on <xref:Npgsql.NpgsqlDataSourceBuilder>
```

Comment:
```suggestion
Considerable effort has gone into reducing Npgsql's size footprint; a minimal Npgsql application using NativeAOT and trimming now takes only around 5MB of disk space. To allow users to achieve a minimal size footprint, <xref:Npgsql.NpgsqlSlimDataSourceBuilder> has been introduced; unlike the standard <xref:Npgsql.NpgsqlDataSourceBuilder>, this builder includes only the very minimum of functionality by default, and allows adding additional features via opt-ins. This allows a pay-for-play approach to application size, where developers can choose only the features they actually need for optimal size. For more information on <xref:Npgsql.NpgsqlDataSourceBuilder>
```

Reply:
No strong feelings here but both terms seem to be used ([some search results](https://www.bing.com/search?q=%22pay-per-play%22+.net+microsoft+site%3Agithub.com&qs=n&form=QBRE&sp=-1&ghc=1&lq=0&pq=%22pay-per-play%22+.net+microsoft+site%3Agithub.com&sc=0-45&sk=&cvid=F9F87E04E6274118AEC0D77CFDF87E9E&ghsh=0&ghacc=0&ghpl=)). I'll leave like that for now but if you feel strongly about it I can change.